tk/environments/tempo/tempo.libsonnet

```diff
@@ -13,6 +13,14 @@ minio + tempo_microservices + tempo_scaling + tempo_tracing + {
       pvc_size: '1Gi',
       pvc_storage_class: 'local-path',
     },
+    tempo_ssb+: {
```

Comment:
What does setting `tempo_ssb` do? I can't find this in any imported mixin.

Reply:
Oh haha, maybe I didn't commit that piece.  I had the notion that we'd have some jsonnet for SSB, but I closed that PR since there was some discussion about maintaining too much deployment code.  Now I'm working on helm, so not sure what the future holds there.    I think the storage limits here are required though, so I'll clean up here.